images/standalone-dns-proxy/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# check=error=true
+
+# Copyright Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG BASE_IMAGE=scratch
+ARG GOLANG_IMAGE=docker.io/library/golang:1.25.3@sha256:6bac879c5b77e0fc9c556a5ed8920e89dab1709bd510a854903509c828f67f96
+
+# BUILDPLATFORM is an automatic platform ARG enabled by Docker BuildKit.
+# Represents the plataform where the build is happening, do not mix with
+# TARGETARCH
+FROM --platform=${BUILDPLATFORM} ${GOLANG_IMAGE} AS builder
+
+# TARGETOS is an automatic platform ARG enabled by Docker BuildKit.
+ARG TARGETOS
+# TARGETARCH is an automatic platform ARG enabled by Docker BuildKit.
+ARG TARGETARCH
+ARG NOSTRIP
+ARG NOOPT
+ARG LOCKDEBUG
+ARG RACE
```

Comment:
The general pattern we use to pass down those variables to the make invocation is to use a single `MODIFIERS` build-arg, see 
https://github.com/cilium/cilium/blob/b0b6b86ba0c0dd0401dfd971402515e53799bd18/images/cilium/Dockerfile#L31-L32 and https://github.com/cilium/cilium/blob/b0b6b86ba0c0dd0401dfd971402515e53799bd18/images/cilium/Dockerfile#L44

That way we don't need to edit every dockerfile when we have a new make variable we want to pass down

Reply:
Make sense, updated it.